src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md

```diff
@@ -13,137 +14,215 @@ date: 2018-11-05
 
-The percent of fast and slow FCP experiences for the most recent release are
-highlighted on top, along with the percent change from the previous release.
-So for example, from August to September 2018, users on this origin experienced
-3.7% more fast FCP and 5.8% fewer slow FCP.
+<img class="w-screenshot" src="./dash_cwv.png" alt="CrUX Dashboard Core Web Vitals overview">
 
-{% Aside %}
-Due to a quirk with Data Studio, you may sometimes see "No Data" here. This is
-normal and due to the previous month's release not being available yet. Datasets
-are released on the second Tuesday of each month.
-{% endAside %}
+Use the Core Web Vitals page to understand how the origin is experienced by 
+desktop and phone users. By default, the most recent month at the time you 
+created the dashboard is selected. To change between older or newer monthly 
+releases, use the "Month" filter at the top of the page.
 
-### Device distribution
+Note that tablet is omitted from these charts by default, but if needed you 
+could remove the "No Tablet" filter in the bar chart configuration, shown 
+below.
 
-The device distribution chart shows you the breakdown of phone, desktop, and
-tablet users over time. Many origins tend to have little to no tablet data so
-you'll see "0%" hanging off the edge of the chart.
+<img class="w-screenshot" src="./dash_cwv_tablet.png" alt="Editing the CrUX Dashboard to show tablets on the Core Web Vitals page">
 
-![CrUX Data Studio device dashboard](./dash_device.png)
+### Metric performance
 
-### Connection distribution
+After the Core Web Vitals page, you'll find standalone pages for all
+[metrics](https://developers.google.com/web/tools/chrome-user-experience-report/#metrics)
+in the CrUX dataset.
 
-The third and final page shows you the breakdown of effective connection types
-per month. There are five possible connection types (4G, 3G, 2G, Slow 2G, and
-Offline), but you will typically only see two or three in the chart. In the
-example above, 2G, Slow 2G, and offline are all 0%.
+<img class="w-screenshot" src="./dash_lcp.png" alt="CrUX Dashboard LCP page">
 
-![CrUX Data Studio effective connection type dashboard](./dash_ect.png)
+Atop each page is the "Device" filter, which you can use to restrict the form 
+factors included in the experience data. For example, you can drill down 
+specifically into phone experiences. This setting persists across pages.
 
-## Sharing the dashboard
+The primary visualizations on these pages are the monthly distributions of
+experiences categorized as "Good", "Needs Improvement", and "Poor". The 
+color-coded legend below the chart indicates the range of experiences included
+in the category. For example, in the screenshot above, you can see the 
+percent of good [Largest Contentful Paint](/lcp/) (LCP) experiences, which are 
+categorized as those less than 2500ms, fluctuating and getting slightly worse 
```

Comment:
I recommend omitting a reference to a specific value (`2500ms`) unless you have some sort of system that tracks every page where specific CWV values are hardcoded (so that you can quickly update all those pages if the values ever change in the future)